### PR TITLE
Update stack.json

### PR DESCRIPTION
**Pull request by alem355**
After Autumn meeting about verifying Tech stack, I updated JSON

### CODE REVIEW

File: static/stack.json [hunk 1]
```diff
@@ -82,208 +82,224 @@
...
+      "Hold": "<code class='lang'>Java</code> <code class='lang'>Objective-C</code>"
     },
     {
       "Area": "QA",
       "Cases": "Functional",
-      "Research": "<code class='tool'>Stubber</code>",
-      "Trial": "<code class='lang'>Kotlin</code> <code class='lang'>Codecept JS</code>",
-      "Best choice": "<code class='lang'>Mocha</code> <code class='tool'>WireMock</code> <code class='lang'>Java</code> <code class='platform'>Umbrella</code> <code class='lang'>TestNG</code> <code class='tool'>Mountebank</code> <code class='lang'>JavaScript</code> <code class='tool'>Allure</code> <code class='lang'>Appium</code> <code class='tool'>Grafana</code> <code class='lang'>Selenide</code>",
+      "Research": "<code class='lang'>Stubber</code>",
+      "Trial": "<code class='lang'>Kotlin</code> <code class='lang'>Codecept JS_</code>",
```
Comment (voborgus) [inline static/stack.json:190]: bug here

File: static/stack.json [hunk 2]
```diff
@@ -82,208 +82,224 @@
...
+      "Best choice": "<code class='lang'>Mocha_</code> <code class='lang'>WireMock</code> <code class='lang'>Java</code> <code class='lang'>Umbrella</code> <code class='lang'>TestNG</code> <code class='lang'>Mountebank</code> <code class='lang'>JavaScript</code> <code class='lang'>Allure</code> <code class='lang'>Appium</code> <code class='lang'>Grafana</code> <code class='lang'>Selenide</code>",
```
Comment (voborgus) [inline static/stack.json:191]: bug here

**Review by aleksandrregner — APPROVED** [review]

File: static/stack.json [hunk 3]
```diff
@@ -82,208 +82,224 @@
...
+      "Hold": "<code class='lang'>Selenium Grid</code> <code class='lang'>Confluence</code>"
     },
     {
       "Area": "API management",
       "Cases": "Protocols & integration agreements",
       "Research": "",
       "Trial": "<code class='lang'>GRPC</code> <code class='lang'>GraphQL</code>",
       "Best choice": "<code class='lang'>HTTP</code> <code class='lang'>HTTP2</code> <code class='lang'>WebSocket</code>",
-      "Hold": "<code class='lang'>SOAP</code> <code class='lang'>FTPS/SFTP</code>"
+      "Hold": "<code class='lang'>SOAP</code> <code class='lang'>FTPS /SFTP</code>"
```
Comment (voborgus) [inline static/stack.json:216]: space not needed here

File: static/stack.json [hunk 4]
```diff
@@ -82,208 +82,224 @@
...
+      "Hold": "<code class='lang'>JMS</code> <code class='lang'>ActiveMQ</code>"
     },
     {
       "Area": "Observability",
       "Cases": "Monitoring (application) ",
       "Research": "",
-      "Trial": "",
-      "Best choice": "<code class='tool'>Prometheus</code> <code class='platform'>Dynatrace</code>",
-      "Hold": "<code class='platform'>Pingdom</code> <code class='tool'>ELK</code> <code class='tool'>Zabbix</code>"
+      "Trial": "VictoriaMetrics",
```
Comment (voborgus) [inline static/stack.json:238]: bug here

File: static/stack.json [hunk 5]
```diff
@@ -82,208 +82,224 @@
...
-      "Best choice": "<code class='tool'>AMQP (RabbitMQ)</code> <code class='platform'>Kafka</code>",
-      "Hold": "<code class='lang'>JMS</code> <code class='tool'>ActiveMQ</code>"
+      "Research": "<code class='lang'>Pulsar</code>",
+      "Trial": "<code class='lang'>NATS/streaming</code> <code class='lang'>Kafka Streams / Connect</code>",
+      "Best choice": "<code class='lang'>AMQP (RabbitMQ)</code> <code class='lang'>Kafka</code>",
+      "Hold": "<code class='lang'>JMS</code> <code class='lang'>ActiveMQ</code>"
     },
     {
       "Area": "Observability",
       "Cases": "Monitoring (application) ",
```
Comment (voborgus) [inline static/stack.json:236]: space not needed here

File: static/stack.json [hunk 6]
```diff
@@ -82,208 +82,224 @@
...
+      "Best choice": "<code class='lang'>Prometheus</code> <code class='lang'>Dynatrace</code>",
+      "Hold": "<code class='lang'>Pingdom</code> <code class='lang'>ELK</code> <code class='lang'>Zabbix</code>"
     },
     {
       "Area": "Observability",
       "Cases": "Alerting",
-      "Research": "<code class='tool'>Statuspage</code>",
-      "Trial": "",
+      "Research": "<code class='platform'>Statuspage</code>",
+      "Trial": "Statuspage",
```
Comment (voborgus) [inline static/stack.json:246]: bug here

File: static/stack.json [hunk 7]
```diff
@@ -292,56 +308,56 @@
...
       "Hold": ""
     },
     {
       "Area": "Infrastructure",
       "Cases": "Cloud",
-      "Research": "<code class='platform'>Mail.ru</code>",
-      "Trial": "<code class='platform'>OpenStack</code>",
-      "Best choice": "<code class='platform'>Yandex Cloud</code>",
-      "Hold": "<code class='platform'>Huawei cloud</code>"
+      "Research": "<code class='platform'>Mail.ru</code> code class='platform'>SberCloud Advance</code>",
```
Comment (voborgus) [inline static/stack.json:341]: bug here

**Review by voborgus — CHANGES_REQUESTED** [review]
can you check please: what you considered as a `language` are not languages itself :(

File: static/stack.json [hunk 8]
```diff
@@ -115,134 +123,142 @@
...
-      "Trial": "<code class='tool'>Yandex.Tank</code> <code class='tool'>HP Performance centre</code>",
+      "Trial": "<code class='tool'>Yandex.Tank</code>",
       "Best choice": "<code class='lang'>Scala</code> <code class='tool'>Gatling</code> <code class='tool'>Lighthouse</code>",
       "Hold": "<code class='tool'>Artillery</code> <code class='tool'>Jmeter</code>"
     },
     {
       "Area": "QA",
       "Cases": "Infrastructure & Documentation",
-      "Research": "",
+      "Research": "<code class='lang'>Allure Server</code>",
```
Comment (voborgus) [inline static/stack.json:205]: it's not a lang

File: static/stack.json [hunk 9]
```diff
@@ -115,134 +123,142 @@
...
-      "Best choice": "<code class='lang'>Mocha</code> <code class='tool'>WireMock</code> <code class='lang'>Java</code> <code class='platform'>Umbrella</code> <code class='lang'>TestNG</code> <code class='tool'>Mountebank</code> <code class='lang'>JavaScript</code> <code class='tool'>Allure</code> <code class='lang'>Appium</code> <code class='tool'>Grafana</code> <code class='lang'>Selenide</code>",
-      "Hold": "<code class='lang'>Selenuim</code> <code class='lang'>Rest-Assured</code> <code class='lang'>Gemini</code> <code class='lang'>Nightwatch</code>"
+      "Research": "<code class='lang'>Stubber</code>",
+      "Trial": "<code class='lang'>Kotlin</code> <code class='lang'>Codecept JS_</code>",
+      "Best choice": "<code class='tool'>Mocha_</code> <code class='tool'>WireMock</code> <code class='lang'>Java</code> <code class='tool'>Umbrella</code> <code class='tool'>TestNG</code> <code class='tool'>Mountebank</code> <code class='lang'>JavaScript</code> <code class='tool'>Allure</code> <code class='tool'>Appium</code> <code class='tool'>Grafana</code> <code class='tool'>Selenide</code>",
```
Comment (voborgus) [inline static/stack.json:191]: bug in Mocha

File: static/stack.json [hunk 10]
```diff
@@ -115,134 +123,142 @@
...
       "Best choice": "<code class='lang'>JavaScript</code> <code class='lang'>React</code>",
-      "Hold": "<code class='lang'>php</code> <code class='platform'>AEM</code> <code class='platform'>Bitrix</code>"
+      "Hold": "<code class='lang'>php</code> <code class='tool'>AEM</code> <code class='tool'>Bitrix</code>"
     },
     {
       "Area": "Frontends",
       "Cases": "Mobile",
-      "Research": "",
-      "Trial": "<code class='lang'>React Native</code> <code class='tool'>LMUI</code>",
+      "Research": "Flutter",
```
Comment (voborgus) [inline static/stack.json:181]: text without tags

File: static/stack.json [hunk 11]
```diff
@@ -292,7 +308,7 @@
       "Cases": "CD tool/framework",
       "Research": "",
       "Trial": "",
-      "Best choice": "<code class='tool'>Ansible</code> <code class='tool'>IaC</code> <code class='tool'>Helm</code> <code class='tool'>Terraform</code>",
+      "Best choice": "<code class='tool'>Ansible</code> <code class='lang'>IaC</code> <code class='lang'>tool</code> <code class='tool'>Terraform</code>",
```
Comment (voborgus) [inline static/stack.json:311]: where is Helm now?

File: static/stack.json [hunk 12]
```diff
@@ -115,134 +123,142 @@
...
+      "Hold": "<code class='lang'>SOAP</code> <code class='lang'>FTPS /SFTP</code>"
     },
     {
       "Area": "API management",
       "Cases": "Gateways & mediation",
       "Research": "<code class='tool'>Kong</code> <code class='tool'>Istio</code>",
       "Trial": "",
       "Best choice": "<code class='platform'>ApiGee</code>",
-      "Hold": "<code class='tool'>Spring Gateway</code> <code class='platform'>Web methods ESB</code> <code class='tool'>Transfile perl</code> <code class='tool'>Spring Zuul</code>"
+      "Hold": "<code class='tool'>ZUUL</code> <code class='platform'>Web methods ESB</code> <code class='tool'>Transfile perl</code>"
```
Comment (voborgus) [inline static/stack.json:224]: where is Spring Gateway now? It's not the same with ZUUL

File: static/stack.json [hunk 13]
```diff
@@ -115,134 +123,142 @@
...
+      "Trial": "<code class='lang'>Kotlin</code> <code class='lang'>Codecept JS_</code>",
+      "Best choice": "<code class='tool'>Mocha_</code> <code class='tool'>WireMock</code> <code class='lang'>Java</code> <code class='tool'>Umbrella</code> <code class='tool'>TestNG</code> <code class='tool'>Mountebank</code> <code class='lang'>JavaScript</code> <code class='tool'>Allure</code> <code class='tool'>Appium</code> <code class='tool'>Grafana</code> <code class='tool'>Selenide</code>",
+      "Hold": "<code class='tool'>Selenuim</code> <code class='lang'>Rest-Assured</code> <code class='lang'>Gemini</code> <code class='lang'>Nightwatch</code>"
     },
     {
       "Area": "QA",
       "Cases": "Performance",
       "Research": "",
-      "Trial": "<code class='tool'>Yandex.Tank</code> <code class='tool'>HP Performance centre</code>",
+      "Trial": "<code class='tool'>Yandex.Tank</code>",
```
Comment (voborgus) [inline static/stack.json:198]: where is HP Performance?

File: static/stack.json [hunk 14]
```diff
@@ -115,134 +123,142 @@
...
+      "Best choice": "<code class='tool'>Camunda</code>",
       "Hold": ""
     },
     {
       "Area": "Business applications backend",
       "Cases": "Data storage",
-      "Research": "<code class='tool'>Ignite</code> <code class='tool'>Tarantool</code>",
-      "Trial": "<code class='tool'>Cassandra</code> <code class='tool'>Redis</code>",
-      "Best choice": "<code class='tool'>MongoDB</code> <code class='tool'>ClickHouse</code> <code class='tool'>PostgeSQL</code> <code class='tool'>S3</code>",
+      "Research": "<code class='platform'>Huawei S3</code>",
```
Comment (voborgus) [inline static/stack.json:149]: where are Ignite and Tarantool?

File: static/stack.json [hunk 15]
```diff
@@ -82,26 +82,34 @@
...
+      "Best choice": "<code class='platform'>Greenplum</code>",
+      "Hold": "<code class='platform'>SQL Server</code>"
+    },
+    {
+      "Area": "Data",
+      "Cases": "DWH operational",
+      "Research": "<code class='tool'>Clickhouse</code>",
+      "Trial": "",
+      "Best choice": "",
+      "Hold": ""
```
Comment (voborgus) [inline static/stack.json:96]: maria DB in the hold?

File: static/stack.json [hunk 16]
```diff
@@ -115,134 +123,142 @@
...
+      "Trial": "Statuspage",
       "Best choice": "<code class='platform'>Opsgenie</code>",
-      "Hold": "<code class='tool'>ELK</code> <code class='tool'>Grafana</code>"
+      "Hold": "<code class='tool'>ELK</code> <code class='tool'>Grafana</code> <code class='tool'>Prometheus</code>"
     },
     {
       "Area": "Observability",
       "Cases": "Tracing",
-      "Research": "<code class='platform'>Open Telemetry</code>",
+      "Research": "<code class='lang'>Open Telemetry</code>",
```
Comment (voborgus) [inline static/stack.json:253]: it's not a lang

File: static/stack.json [hunk 17]
```diff
@@ -115,134 +123,142 @@
...
-      "Hold": "<code class='lang'>php</code> <code class='platform'>AEM</code> <code class='platform'>Bitrix</code>"
+      "Hold": "<code class='lang'>php</code> <code class='tool'>AEM</code> <code class='tool'>Bitrix</code>"
     },
     {
       "Area": "Frontends",
       "Cases": "Mobile",
-      "Research": "",
-      "Trial": "<code class='lang'>React Native</code> <code class='tool'>LMUI</code>",
+      "Research": "Flutter",
+      "Trial": "<code class='lang'>React Native</code> <code class='lang'>LMUI</code> <code class='lang'>Kotlin Multiplatform</code> <code class='lang'>Swift</code>",
```
Comment (voborgus) [inline static/stack.json:182]: Swift duplicated

File: static/stack.json [hunk 18]
```diff
@@ -115,134 +123,142 @@
...
+      "Best choice": "<code class='platform'>Platformeco</code>",
       "Hold": "<code class='lang'>Python</code>"
     },
     {
       "Area": "Business applications backend",
       "Cases": "Complex BL, Process orchestration",
-      "Research": "<code class='platform'>Drools</code>",
-      "Trial": "<code class='platform'>Java state machine</code>",
-      "Best choice": "<code class='platform'>Camunda</code>",
+      "Research": "",
```
Comment (voborgus) [inline static/stack.json:141]: Drools to hold?

Comment (alem355) [reply static/stack.json:141]: вообще убрали

File: static/stack.json (hunk 14 above)
Comment (alem355) [reply static/stack.json:149]: There is no Ignite anymore

File: static/stack.json (hunk 13 above)
Comment (alem355) [reply static/stack.json:198]: Added

File: static/stack.json [hunk 19]
```diff
@@ -116,33 +124,41 @@
       "Cases": "CRUD",
       "Research": "",
       "Trial": "<code class='lang'>GO</code> <code class='platform'>Platformeco</code> <code class='lang'>NodeJS</code> <code class='lang'>NEST JS</code> <code class='lang'>Typescript</code> <code class='lang'>Python (Django)</code> <code class='lang'>Python (fastapi)</code> <code class='lang'>Python (flask)</code>",
-      "Best choice": "<code class='lang'>JS</code> <code class='lang'>Java</code> <code class='lang'>Springboot</code> <code class='lang'>Kotlin</code>",
+      "Best choice": "<code class='lang'>Node.JS</code> <code class='lang'>Java</code> <code class='lang'>Springboot</code> <code class='lang'>Kotlin</code>",
```
Comment (voborgus) [inline static/stack.json:127]: NodeJS Duplicate

File: static/stack.json (hunk 14 above)
Comment (voborgus) [reply static/stack.json:149]: that's mean it should go to hold, am I right?

File: static/stack.json [hunk 20]
```diff
@@ -152,41 +168,41 @@
...
     {
-      "Area": "Frontends",
-      "Cases": "Mobile",
-      "Research": "",
-      "Trial": "<code class='lang'>React Native</code> <code class='tool'>LMUI</code>",
-      "Best choice": "<code class='lang'>Swift</code> <code class='lang'>Kotlin</code>",
-      "Hold": ""
+        "Area": "Frontends",
+        "Cases": "Mobile",
+        "Research": "<code class='lang'>Flutter</code>",
```
Comment (voborgus) [inline static/stack.json:181]: platform

File: static/stack.json [hunk 21]
```diff
@@ -152,41 +168,41 @@
...
     },
     {
       "Area": "QA",
       "Cases": "Functional",
-      "Research": "<code class='tool'>Stubber</code>",
+      "Research": "<code class='tool'>Stubber</code>  <code class='tool'>Allure Server</code>",
       "Trial": "<code class='lang'>Kotlin</code> <code class='lang'>Codecept JS</code>",
-      "Best choice": "<code class='lang'>Mocha</code> <code class='tool'>WireMock</code> <code class='lang'>Java</code> <code class='platform'>Umbrella</code> <code class='lang'>TestNG</code> <code class='tool'>Mountebank</code> <code class='lang'>JavaScript</code> <code class='tool'>Allure</code> <code class='lang'>Appium</code> <code class='tool'>Grafana</code> <code class='lang'>Selenide</code>",
+      "Best choice": "<code class='lang'>Mocha</code> <code class='lang'>Java</code> <code class='lang'>JavaScript</code> <code class='tool'>WireMock</code> <code class='lang'>Java</code> <code class='platform'>Umbrella</code> <code class='lang'>TestNG</code> <code class='tool'>Mountebank</code> <code class='lang'>JavaScript</code> <code class='lang'>Appium</code> <code class='tool'>Grafana</code> <code class='lang'>Selenide</code>",
```
Comment (voborgus) [inline static/stack.json:191]: duplicates here

File: static/stack.json (hunk 13 above)
Comment (voborgus) [reply static/stack.json:198]: where?

File: static/stack.json [hunk 22]
```diff
@@ -80,19 +80,27 @@
       "Hold": ""
     },
     {
-      "Area": "Data",
-      "Cases": "DWH",
-      "Research": "<code class='lang'>Clickhouse</code>",
-      "Trial": "<code class='platform'>Maria DB</code>",
-      "Best choice": "<code class='lang'>Greenplum</code>",
-      "Hold": "<code class='lang'>SQL Server</code>"
+       "Area": "Data",
```
Comment (voborgus) [inline static/stack.json:83]: wrong spaces at the start of the line

File: static/stack.json [hunk 23]
```diff
@@ -152,41 +168,41 @@
       "Hold": "<code class='platform'>ESB</code> <code class='tool'>DB sync</code>"
     },
     {
-      "Area": "Frontends",
-      "Cases": "Web",
-      "Research": "",
-      "Trial": "<code class='lang'>TypeScript</code> <code class='lang'>Vue</code> <code class='lang'>Svelte</code> <code class='lang'>WebComponents</code> <code class='lang'>Angular</code> <code class='lang'>OKAPI</code>",
-      "Best choice": "<code class='lang'>JavaScript</code> <code class='lang'>React</code>",
-      "Hold": "<code class='lang'>php</code> <code class='platform'>AEM</code> <code class='platform'>Bitrix</code>"
+        "Area": "Frontends",
```
Comment (voborgus) [inline static/stack.json:171]: wrong spaces at the start of the line

**Review by voborgus — APPROVED** [review]
Now it's fine. Thanks @alem355

**Pull request merged**